tldw_Server_API/app/api/v1/endpoints/data_tables.py

```
@@ -0,0 +1,1053 @@
+        return await asyncio.to_thread(adapter.export, structured, format=export_format)
+    return adapter.export(structured, format=export_format)
+
+
+async def _wait_for_job_completion(
+    jm: JobManager,
+    job_id: int,
+    *,
+    timeout_seconds: int = 300,
+    poll_interval: float = 1.0,
+) -> Dict[str, Any]:
+    deadline = time.monotonic() + timeout_seconds
```

Ruff (0.14.13)

Undefined name `time`

(F821)